pkg/api/v1/openapi_generated.go

```diff
@@ -156,6 +156,20 @@ func schema_kubevirt_pkg_api_v1_CPU(ref common.ReferenceCallback) common.OpenAPI
 							Format:      "int64",
 						},
 					},
+					"sockets": {
+						SchemaProps: spec.SchemaProps{
+							Description: "Sockets specifies the number of sockets inside the vmi. Must be a value greater or equal 1.",
+							Type:        []string{"integer"},
+							Format:      "int64",
```

Comment:
I wonder if we should use unsigned integer. Not sure what the kubevirt best practices are.